### PR TITLE
Feature/add server and autoscaling nodes datasource

### PR DESCRIPTION
**Pull request by duyleekun**
I need those 2 data sources to get server detail of a node in autoscaling group in terraform

### CODE REVIEW

File: bizflycloud/datasource_bizflycloud_autoscaling_nodes.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	}
+
+	if okId {
+		nodesResult := make([]map[string]interface{}, len(osNodes))
+		for i, node := range osNodes {
+			nodesResult[i] = map[string]interface{}{
+				"name":         node.Name,
+				"id":           node.ID,
+				"profile_name": node.ProfileName,
+				//"addresses":     node.Addresses,
```
Comment (greatbn) [inline bizflycloud/datasource_bizflycloud_autoscaling_nodes.go:62]: Do we need to remove this comment ?

**Review by greatbn — CHANGES_REQUESTED** [review]

File: bizflycloud/datasource_bizflycloud_autoscaling_nodes.go (hunk 1 above)
Comment (duyleekun) [reply bizflycloud/datasource_bizflycloud_autoscaling_nodes.go:62]: I want to be consistent with the SDK but setting it this way doesn't work. Can we keep this comment as a reminder so someone else can tackle this later?

Comment (greatbn) [reply bizflycloud/datasource_bizflycloud_autoscaling_nodes.go:62]: so we should add TODO to the comment.

**Review by greatbn — APPROVED** [review]

**Comment by duyleekun** [comment]
@greatbn I remembered that for this to work we need to bump the gobizfly to newest version which contains this fix

https://github.com/bizflycloud/gobizfly/pull/112

Should I create a PR to bump dependencies? I don't know what's the policy to bump lib ver.

**Pull request merged**